interfaces/builtin/hidraw_test.go

```diff
@@ -232,21 +233,30 @@ SUBSYSTEM=="hidraw", SUBSYSTEMS=="usb", ATTRS{idVendor}=="ffff", ATTRS{idProduct
 }
 
 func (s *HidrawInterfaceSuite) TestConnectedPlugAppArmorSnippets(c *C) {
-	expectedSnippet1 := []byte(`/dev/hidraw0 rw,
-`)
-	snippet, err := s.iface.ConnectedPlugSnippet(s.testPlugPort1, s.testSlot1, interfaces.SecurityAppArmor)
+	expectedSnippet1 := `/dev/hidraw0 rw,
```

Comment:
Ah, nvm. You're of course right, I missed the earlier comment about hidraw interface itself. After fixing hidraw all these newlines need to go away indeed.